packages/react/src/components/Chip/_ChipButton/ChipButton.tsx

```diff
@@ -0,0 +1,53 @@
+import type { HTMLAttributes } from 'react';
+import React, { forwardRef } from 'react';
+import cn from 'classnames';
+
+import { Paragraph } from '../../Typography';
+import type { OverridableComponent } from '../../../types/OverridableComponent';
+
+import classes from './ChipButton.module.css';
+
+export type ChipButtonProps = {
+  /**
+   * Changes padding and font-sizes.
+   */
+  size?: 'xsmall' | 'small';
+  /**
+   * Toggles `aria-pressed` and visual-changes
```

Comment:
Hvordan ser en konsument av komponent at chipen er checked/uchecked når den ikke har en prop som sier noe om checked-staten til den?

Reply:
Eller ser den har en prop selected nå men denne vises ikke i storybook?

<img width="1281" alt="image" src="https://github.com/digdir/designsystem/assets/31471142/3f0bd962-1ea9-47ed-ab18-07de519df00b">

Reply:
Er ikke det den selected-propen?

Reply:
Huh.. Den gjør det for meg 😅

![image](https://github.com/digdir/designsystem/assets/22012997/9b43dbb2-abf1-4c9c-9369-23e5b01c7d43)

Reply:
Ahh ja måtte kjøre yarn build